Wikipedia/Code/WebViewController.m

```diff
@@ -290,8 +290,7 @@ - (WMFArticleFooterView*)footerLicenseView {
         @weakify(self);
         [_footerLicenseView.showLicenseButton bk_addEventHandler:^(id sender) {
             @strongify(self);
-            MWKSite* site = [[MWKSite alloc] initWithDomain:WMFDefaultSiteDomain language:@"en"];
-            [self.delegate webViewController:self didTapOnLinkForTitle:[site titleWithString:WMFLicenseTitleOnENWiki]];
```

Comment:
Just a though,
'@"by-sa/3.0";'
and
`MWKSite* site = [[MWKSite alloc] initWithDomain:WMFDefaultSiteDomain language:@"creativecommons:"];
[self.delegate webViewController:self didTapOnLinkForTitle:[site titleWithString:WMFLicenseTitleOnENWiki]];`
would not work then?